mlos_bench/mlos_bench/config/schemas/environments/common-environment-subschemas.json

```diff
@@ -82,6 +87,48 @@
                 }
             ]
         },
+
+        "shell_param_type": {
+            "description": "All parameters must be valid shell variable names, else use shell_params_mapping.",
+            "type": "string",
+            "pattern": "^[a-zA-Z_][a-zA-Z0-9_]+$"
+        },
+        "shell_params_config": {
+            "type": "object",
+            "properties": {
+                "shell_params_match": {
```

Comment:
@motus, since this isn't supported, we should remove it.

Reply:
Although an example of ones that might be handy to include: `AZURE_*` environment variables.  In fact these could be setup in the shell environment, slurped into the `mlos_bench` globals automatically, and then passed along to the remote/local env for scripts which would be super convenient.